### ISSUE TITLE
It works but it is not possible to deploy with my standalone app

### ISSUE THREAD

**Issue by bakikucukcakiroglu** [montalvomiguelo/theme-extension-vite#2]
Both ```pnpm run dev``` and```pnpm run deploy``` works on this template. However, how can I use this extension in my standalone app? Can you please explain the workflow I follow with this repository? I tried to copy extensions folder in the template after developing and building, actually it worked but counter buttons, counter number vs(things that is not in snippets) doesn't appear. They appear if I directly use your template but not if I copy the builded files.

**Comment by bakikucukcakiroglu** [comment]
<img width="640" alt="Screenshot 2024-07-24 at 02 51 10" src="https://github.com/user-attachments/assets/084776ac-ecde-43d7-b8bb-cf3d2236daf3">

I crated two extensions. One from this repo directly. For the other I copied extensions folder of this repo after building to my repo. I added both of them to my development store to test. Even if we expect them to look the same, deploy of this repo works while mine fails. (Actually mine fails when it is alone while deploy of this repo works even alone. However, if I add them together they both work. Probably because missing scripts and styling come with the deploy of this repo. 

In the photo I uploaded, the top section corresponds to mine. And the bottom corresponds to this repo. The difference is ```src``` of scripts. How to configure them?
